configs/cornernet/README.md

```diff
@@ -0,0 +1,28 @@
+# CornerNet
+
+## Introduction
+```
+@inproceedings{law2018cornernet,
+  title={Cornernet: Detecting objects as paired keypoints},
+  author={Law, Hei and Deng, Jia},
+  booktitle={15th European Conference on Computer Vision, ECCV 2018},
+  pages={765--781},
+  year={2018},
+  organization={Springer Verlag}
+}
+```
+
+## Results and models
+
+| Backbone        | Batch Size | Step/Total Epochs | Mem (GB) | Inf time (fps) | box AP | Pytorch Version | Download |
+| :-------------: | :--------: |:----------------: | :------: | :------------: | :----: | :-------------: | :------: |
+| HourglassNet-104 | 5 x 10 | 180/210 | 13.9 | | 41.3 | 1.5 | |
+| HourglassNet-104 | 6 x 8 | 180/210 | 15.9 | | 41.2 | 1.5 | |
+| HourglassNet-104 | 6 x 16 | 180/210 | 15.9 | | 41.1 | 1.5 | |
+| HourglassNet-104 | 3 x 32 | 180/210 | 9.5 | | 40.4 | 1.3 | |
+
+Note: 
+- All models are tested with TTA (single-scale and `flip=True`).
```

Comment:
To have a fair comparison with other models, we need also report single-scale testing results.